chia/consensus/blockchain.py

```diff
@@ -795,11 +799,21 @@ async def pre_validate_blocks_multiprocessing(
         self,
         blocks: List[FullBlock],
         npc_results: Dict[uint32, NPCResult],  # A cache of the result of running CLVM, optional (you can use {})
+        sub_slot_iters: Optional[uint64] = None,
+        difficulty: Optional[uint64] = None,
+        prev_ses_block: Optional[BlockRecord] = None,
```

Comment:
what would you think about putting these after the `*,` to require them to be named in calls?
It would make calls a little bit more clear I think